### PR TITLE
feature/ATC-192

### PR DESCRIPTION
**Pull request by Dabea**
completes n8rzz/atc#192 

formatted white space on variables and in display text.

I have found a few things in `AircraftInstanceModel` that should help with spacing. I searched though many other files but did not see anything else. If you have any other files that might be a good candidate to be searched though let me know.

### CODE REVIEW

**Comment by n8rzz** [comment]
I'd like for mr @erikquinn to look at this one.  there are _a lot_ of strings that get sent to the UI and, although I know where most of them are, I'd like for him to be the final word on this PR. 😉 

_note: as you can see above, I spun up a PR instance for this one.  This will auto-deploy on push, so it should make testing and vetting this easier_

File: src/assets/scripts/aircraft/AircraftInstanceModel.js
Comment (erikquinn) [inline src/assets/scripts/aircraft/AircraftInstanceModel.js]: Further down in this function, this variable gets used in ``` cs += ` ${radio_spellOut(callsign)} ${heavy}`; ```. If the aircraft is neither "heavy" nor "super", then `heavy` will remain an empty string, which would yield a second space in the returned callsign.

Now that I think about it, it's only using this to speak the callsign aloud, so it actually shouldn't really matter.

**Review by erikquinn — CHANGES_REQUESTED** [review]
The only other problems I'm seeing space-wise are:

 - trailing space at `AircraftInstanceModel.js:939` needs to be removed
 - `AircraftInstanceModel.runAltitude()` should be amended such that its returns do not end with a space when `expedite` is an empty string

Udda den DAT, every ting A-OK!
^ ([in case you don't get the reference](https://www.youtube.com/watch?v=_IqWal_EmBg) :wink:)

**Comment by n8rzz** [comment]
@Dabea updated with #220 file structure. be sure to re-pull and `npm install` before continuing work on this one.

**Comment by Dabea** [comment]
Looks Like i did not push my last changes before the update. This will address the extra space until the expedite method is added.

Also should I add these changes under minor changes for 3.3.0 as well?

**Comment by n8rzz** [comment]
@Dabea yes please. v3.3.0 only please.

**Comment by Dabea** [comment]
This one is complete and  can be merged (unless i am missing something)

**Review by erikquinn — APPROVED** [review]

**Pull request merged**
